### PR TITLE
INT-3325 upgrade to iq 1.97.0

### PR DESCRIPTION
**Pull request by jflinchbaugh**
automatically generated by script.

JIRA: https://issues.sonatype.org/browse/INT-3325

### CODE REVIEW

**Comment by jflinchbaugh** [comment]
This is the first time the change has been generated completely by script for IQ. After this, I may suggest we not bother reviewing.

File: build/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
+# DO NOT MODIFY. This is produced by template.
```
Comment (kellyrob99) [inline build/Dockerfile:1]: did we add this or is it coming from tooling?

Comment (jflinchbaugh) [reply build/Dockerfile:1]: our templating script adds it to each file it produces.

Comment (kellyrob99) [reply build/Dockerfile:1]: I see where it's coming from now. Suggest that this should be updated to include the name of the script to run and instead say something like `DO NOT MODIFY BY HAND. Run scripts/new_version.sh to apply any needed modifications`

Comment (jflinchbaugh) [reply build/Dockerfile:1]: Some of these files are artifacts that end-users may see. I was reluctant to refer to specific build details that won't make any sense to them. Should that be a concern?

**Pull request merged**